# Examining Racial Discrimination in the US Job Market

### Background
Racial discrimination continues to be pervasive in cultures throughout the world. Researchers examined the level of racial discrimination in the United States labor market by randomly assigning identical résumés to black-sounding or white-sounding names and observing the impact on requests for interviews from employers.

### Data
In the dataset provided, each row represents a resume. The 'race' column has two values, 'b' and 'w', indicating black-sounding and white-sounding. The column 'call' has two values, 1 and 0, indicating whether the resume received a call from employers or not.

Note that the 'b' and 'w' values in race are assigned randomly to the resumes when presented to the employer.

<div class="span5 alert alert-info">
### Exercises
You will perform a statistical analysis to establish whether race has a significant impact on the rate of callbacks for resumes.

Answer the following questions **in this notebook below and submit to your Github account**. 

   1. What test is appropriate for this problem? Does CLT apply?
   2. What are the null and alternate hypotheses?
   3. Compute margin of error, confidence interval, and p-value.
   4. Write a story describing the statistical significance in the context or the original problem.
   5. Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis?

You can include written notes in notebook cells using Markdown: 
   - In the control panel at the top, choose Cell > Cell Type > Markdown
   - Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet


#### Resources
+ Experiment information and data source: http://www.povertyactionlab.org/evaluation/discrimination-job-market-united-states
+ Scipy statistical methods: http://docs.scipy.org/doc/scipy/reference/stats.html 
+ Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet
</div>
****

In [6]:
import pandas as pd
import numpy as np
from scipy import stats

In [7]:
data = pd.io.stata.read_stata('data/us_job_market_discrimination.dta')

In [8]:
# number of callbacks for black-sounding names
sum(data[data.race=='b'].call)

157.0

In [9]:
# number of callbacks for white-sounding names
sum(data[data.race == 'w'].call)

235.0

In [10]:
data.groupby('call').size()

call
0.0    4478
1.0     392
dtype: int64

In [11]:
data.head()

id ad  education  ofjobs  yearsexp  honors  volunteer  military  empholes  \
0  b  1          4       2         6       0          0         0         1   
1  b  1          3       3         6       0          1         1         0   
2  b  1          4       1         6       0          0         0         0   
3  b  1          3       4         6       0          1         0         1   
4  b  1          3       3        22       0          0         0         0   

   occupspecific  ...  compreq  orgreq  manuf  transcom  bankreal trade  \
0             17  ...      1.0     0.0    1.0       0.0       0.0   0.0   
1            316  ...      1.0     0.0    1.0       0.0       0.0   0.0   
2             19  ...      1.0     0.0    1.0       0.0       0.0   0.0   
3            313  ...      1.0     0.0    1.0       0.0       0.0   0.0   
4            313  ...      1.0     1.0    0.0       0.0       0.0   0.0   

  busservice othservice  missind  ownership  
0        0.0        0.0      0.0             
1        0.0        0.0      0.0             
2        0.0        0.0      0.0             
3        0.0        0.0      0.0             
4        0.0        1.0      0.0  Nonprofit  

[5 rows x 65 columns]

In [12]:
len(data)

4870

** 1. What test is appropriate for this problem? Does CLT apply? **

Since we are comparing two distinct groups, I would use a 2 sample t-test. This is because we assume that these two groups are independent of each other. This would be different than a paired t-test. A paired t-test is similar to a one sample t-test but it compares pre-treatment and post-treatment of a single group. This means that the post-treatment sample is dependent on the pre-treatment sample. This is why a 2 sample t-test is necessary because it has two independent groups.

CLT does still apply. We have a 4870 observations which is sufficient for the CLT to apply to this dataset. By the CLT, we can use the normal distribution to approximate binomial probabilities.

** 2. What are the null and alternate hypotheses? **

H0: proportion of black calls - proportion of white calls == 0

H1: proportion of black calls - proportion of white calls != 0



**3. Compute margin of error, confidence interval, and p-value. **

In [13]:
black = data.call[data.race == 'b']
white = data.call[data.race == 'w']
stats.ttest_ind(black,white)

Ttest_indResult(statistic=-4.114705290861751, pvalue=3.940802103128886e-05)

In [14]:
xbar_b = black.mean()
sd_b = black.std()    

xbar_w = white.mean()
sd_w = white.std()

sample_statistic = xbar_b - xbar_w
se = (((sd_b)**2/len(black)) + ((sd_w)**2/len(white)))**(1/2)
me = 1.96*se
plus = sample_statistic + me
minus = sample_statistic - me 
print (' 95% Confidence Interval: {minus} to {plus} \n Margin of Error: {me}'.format(minus=minus,plus=plus, me=me))

 95% Confidence Interval: -0.04729127241789561 to -0.016774437292225546 
 Margin of Error: 0.015258417562835034


** 4. Write a story describing the statistical significance in the context or the original problem. **

Racial discrimination is still present in todays society. In the United States, a study was conducted that assigned employers at random resumes of black sound and white sounding names. This was to find if race had an impact on requests for interviews by employers. The data found that there was a statistically significant difference between requests for interviews based on black or white sounding names. Employers were actually more likely to choose white sounding names than black sounding names.

** 5. Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis? **

No, it does not. The reason is because we have only analyzed one variable. You could do a linear regression with multiple demographics variables to see which impacts callback success the most.